unittests/test_rotations.py

```diff
@@ -0,0 +1,125 @@
+"""Unit tests for rotations module."""
+
+import numpy as np
+import unittest
+
+import rotations
+
+
+class TestRotationsMethods(unittest.TestCase):
+
+    def test_regularize_rotation_vector(self):
+        rot_vec_1 = 2.5 * np.pi * np.array([0., 0., 1.])
+        rot_vec_1 = rotations.regularize_rotation_vector(rot_vec_1)
+        rot_vec_1_expected = np.pi / 2. * np.array([0., 0., 1.])
+        self.assertTrue(np.allclose(rot_vec_1, rot_vec_1_expected))
+
+        rot_vec_2 = 1.5 * np.pi * np.array([0., 1., 0.])
+        rot_vec_2 = rotations.regularize_rotation_vector(rot_vec_2)
+        rot_vec_2_expected = np.pi / 2. * np.array([0., -1., 0.])
+        self.assertTrue(np.allclose(rot_vec_2, rot_vec_2_expected))
+
+        rot_vec_3 = 11 * np.pi * np.array([1., 2., 3.])
+        rot_vec_3 = rotations.regularize_rotation_vector(rot_vec_3)
+        fact = 0.84176874548664671 * np.pi / np.sqrt(14)
```

Comment:
where does this constant come from? can you be a bit more clear into the kind of things you're testing here?? Even though I usually hate comments, in unit tests it's fine.